build-core-armv7-imxrt.sh

```diff
@@ -24,18 +19,15 @@ b_log "Building libphoenix"
 
 b_log "Building phoenix-rtos-filesystems"
 (cd phoenix-rtos-filesystems && make $MAKEFLAGS $CLEAN all)
-cp "$PREFIX_PROG_STRIPPED/dummyfs" _build
-cp "$PREFIX_PROG_STRIPPED/meterfs" _build
 
 b_log "Building phoenix-rtos-devices"
 (cd phoenix-rtos-devices && CFLAGS+=$DEVICE_FLAGS make $MAKEFLAGS $CLEAN all)
-cp "$PREFIX_PROG_STRIPPED/imxrt-multi" _build
-cp "$PREFIX_PROG_STRIPPED/imxrt-flash" _build
+
+b_log "Building phoenix-rtos-usb"
+(cd phoenix-rtos-usb && make $MAKEFLAGS $CLEAN all)
 
 b_log "Building coreutils"
 (cd phoenix-rtos-coreutils && make $MAKEFLAGS $CLEAN all)
-cp "$PREFIX_PROG_STRIPPED/psh" _build
-cp "$PREFIX_PROG_STRIPPED/psd" _build
```

Comment:
Why remove those lines?

Reply:
The built binaries get installed to _build/$TARGET/prog(/prog.stripped) directory. In the imxrt case we use the non-stripped binaries, since the RTOS image generating script strips the binaries and saves the debug symbols in _build/gdb_symbols file. the There is no need to create duplicates two folders up the directory tree.